### ISSUE TITLE
DOCUMENTATION

### ISSUE THREAD

**Issue by pdelamora** [TRIQS/cthyb#28]
Dear Oliver and Priyanka,
++++Last time I wrote to you I said;
When I wrote to you last november and you suggested that I should put these issues in the "TRIQS issues" 
I assumed from this answer that the DMFT was already a part of TRIQS. 
It was a surprise for me last week to find that I have to write my own DMFT program based in the languages used in TRIQS.
So it would be nice to have a program example of a simple compound calculated with WIEN2k, I don't know if there is something like that.
++++You answered;
you should know that DMFT is not a black-box method
and then you added
With the solver in hand, the simplest DMFT loop can be written in about 10 lines.
++++++++++++++++
++++++++++++++++
Today I add;
Yes, I understand and I am still not an expert in DMFT, 
but the minimun to start it would be great.
++1- How can one use the WIEN2k output
++2- How to call the CTHYB solver

These issues should be in the DOCUMENTATION;
++A quick introduction to the solver
++The hybridization-expansion algorithm in details
++User Manual
++Implementation details 

but as I said, these folders are empty, 
so it is quite difficult even for a person that knows DMFT to use the TRIQS package

It would be great to have a program to study the NiO Mott insulator, 
but also, for this purpose, Ni or Fe would be sufficient.

**Comment by aichhorn** [comment]
Dear pdelamora,
I am surprised that you say that there is no introduction on how to use TRIQS for DMFT whatsoever. Please got to http://ipht.cea.fr/triqs/1.2/applications/dft_tools/documentation.html where you find a quite complete tutorial on how to do the DFT+DMFT calculation for SrVO3. From these scripts it is quite simple to switch to NiO. Most of the issues you mentioned above are answered in this documentation and tutorial. We are working on a reference manual for the dft_tools which should be online soon.
Regards, Markus

**Comment by pdelamora** [comment]
Dear Markus,
Thank you for your reply, I realized that was confused;
I was not clear what TRIQS was all about, but from your reply I realized that I was looking at the wrong directory. I am still not that clear, but now I know my way forward.

I was looking for LDA+DMFT and Oliver Parcollet told me to look into the solver cthyb.
http://ipht.cea.fr/triqs/1.2/applications/cthyb/
when I entered that page I saw that it had almost no information and nothing on DMFT.
Oliver Parcollet further mentioned that DMFT is a flexible tool that one has to write its own program, later he mentions that there are/will be several complete examples in dft_tools, ready to run.
Unfortunately I did not read the "are/will" carefully and I understood only the "will" so I thought that I should write my own programs from zero, and with this in mind I wrote the last latter.
Now reading your reply I see that there are clear examples --in another directory--, so slowly I can make my way forward.

Markus, thanks again and Oliver Parcollet and Priyanka Seth sorry for all this confusion and thank again for all your help.
